org.jacoco.core/src/org/jacoco/core/internal/flow/Instruction.java

```diff
@@ -25,6 +27,8 @@
 
 	private Instruction predecessor;
 
+	public AbstractInsnNode node;
```

Comment:
As it is always set after the instance is created this should be final and set within the constructor.

For consistency I would prefer to make it private with a public getter.